src/components/App.js

```diff
@@ -175,32 +176,33 @@ export default class App extends React.Component {
         this._player.load();
       }
       this._player.volume = 0;
-      this._player.play();
+      this._player.play().then(() => {
+        this.setState(state => {
+          return {
+            audioConfig: { ...state.audioConfig, currentVolume: 0 },
+            playing: true,
+            pullMeta: true
+          };
+        });
 
-      let audioConfig = this.state.audioConfig;
-      audioConfig.currentVolume = 0;
-      this.setState({
-        audioConfig,
-        playing: true,
-        pullMeta: true
+        this.fadeUp();
       });
-
-      this.fadeUp();
     }
   }
 
   pause() {
-    if (this._player.paused) return;
```

Comment:
`state.playing` should track this properly, so we should never need to check it directly.